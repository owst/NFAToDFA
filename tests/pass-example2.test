0
0--a,b->0
0--b->1
1
