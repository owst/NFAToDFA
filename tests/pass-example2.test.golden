digraph NFA {
rankdir = LR;
node [shape = point]; init
0 [label="0" shape="circle" ]; init -> 0;
0 -> 0 [label="{b, a}"];
1 [label="1" shape="doublecircle" ];
0 -> 1 [label="b"];
}
